kubernetes/ocfweb-web.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ocfweb-web-service
+spec:
+  selector:
+    app: ocfweb-web
+  ports:
+    - port: 80
+      targetPort: 8000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ocfweb-web-deployment
+  labels:
+    app: ocfweb-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ocfweb-web
+  template:
+    metadata:
+      labels:
+        app: ocfweb-web
+    spec:
+      containers:
+        - name: ocfweb-web-deploy
+          image: "docker.ocf.berkeley.edu/ocfweb-web:keur-1"
```

Comment:
This should be changed to the real value before merging, no? (Same for the other files)

Reply:
Yeah thanks